components/skeleton/skeleton-mixin.js

```diff
@@ -0,0 +1,101 @@
+import '../colors/colors.js';
+import { css } from 'lit-element/lit-element.js';
+import { dedupeMixin } from '@open-wc/dedupe-mixin';
+
+export const skeletonStyles = css`
+	@keyframes loadingPulse {
+		0% { background-color: var(--d2l-color-sylvite); }
+		50% { background-color: var(--d2l-color-regolith); }
+		75% { background-color: var(--d2l-color-sylvite); }
+		100% { background-color: var(--d2l-color-sylvite); }
+	}
+	:host([skeleton]) .d2l-skeletize::before {
+		animation: loadingPulse 1.8s linear infinite;
+		background-color: var(--d2l-color-sylvite);
+		border-radius: 0.2rem;
+		bottom: 0;
+		content: '';
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+		z-index: 2000;
```

Comment:
Do we need this z-index to be so high?  I think from @mdgbayly 's previous reply, it sounded like there wasn't a specific reason to choose this value, and that it quite possibly could have been something much smaller, perhaps even as low as 1.

Reply:
Nope, not in my testing. I've set it to `1` as you suggested and we can adjust if it becomes necessary.